tests/migration/Core/V6_4/Migration1663402842AddPathToMediaTest.php

```diff
@@ -0,0 +1,47 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Migration\Core\V6_4;
+
+use Doctrine\DBAL\Connection;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\Migration\IndexerQueuer;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelLifecycleManager;
+use Shopware\Core\Migration\V6_4\Migration1663402842AddPathToMedia;
+use Shopware\Tests\Migration\MigrationTestTrait;
+
+/**
+ * @internal
+ *
+ * @covers \Shopware\Core\Migration\V6_4\Migration1663402842AddPathToMedia
+ */
+class Migration1663402842AddPathToMediaTest extends TestCase
+{
+    use MigrationTestTrait;
+
+    private Connection $connection;
+
+    private IndexerQueuer $queuer;
+
+    protected function setUp(): void
+    {
+        $this->connection = KernelLifecycleManager::getConnection();
+        $this->queuer = new IndexerQueuer($this->connection);
+        // remove the media folder indexer from the queue, if it may be already added by some other migration
+        $this->queuer->finishIndexer(['media.indexer']);
+    }
+
+    public function testTablesHaveFieldPath(): void
+    {
+        $migration = new Migration1663402842AddPathToMedia();
+        $migration->update($this->connection);
```

Comment:
You should drop the new column before this migration to check if this gets created

Reply:
Also you should call the migration twice to be sure that nothing breaks